models/Fields.js

```diff
@@ -48,6 +48,7 @@ const ngoFields = [
   "bbmp",
   "nov",
   "covid19",
+  "vcode"
```

Comment:
this isn't required as the value comes through url params

Reply:
This is required as we will save the generated code while saving the NGO registration form details itself.